hardware_interface/include/hardware_interface/component_interfaces/actuator_interface.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 ROS2-Control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef HARDWARE_INTERFACE__COMPONENT_INTERFACES__ACTUATOR_INTERFACE_HPP_
+#define HARDWARE_INTERFACE__COMPONENT_INTERFACES__ACTUATOR_INTERFACE_HPP_
+
+#include <algorithm>
+#include <array>
+#include <string>
+#include <vector>
+
+#include "hardware_interface/component_info.hpp"
+#include "hardware_interface/types/hardware_interface_return_values.hpp"
+#include "hardware_interface/visibility_control.h"
+
+namespace hardware_interface
+{
+
+class ActuatorInterface
```

Comment:
We should consider interface from  54165d3

Reply:
can you elaborate on this? Not sure I follow

Reply:
I meant clams and ˋset_from_hardware ˋ and ˋset_from_controllerˋ function. Then we would call transmissions if needed. Or should transmissions be placed somewhere else?

Reply:
transmissions are pretty orthogonal at this point I would say. 

I could see that we specify `ComponentsInfo` further or specialize them for actuators which contain transmission information.

Reply:
Ok, so we would than call it using ComponentsInfo. There is a question if we are using this class to store data, how should class now who is writing, hardware or controller. Or this is nor use-case, but then we would probably manage this in the Container

Reply:
This will be discussed in ros-controls/ros2_control#101